src/lib/Components/Bidding/Screens/Registration.tsx

```diff
@@ -142,8 +132,12 @@ export class Registration extends React.Component<RegistrationProps, Registratio
 
     commitMutation(this.props.relay.environment, {
       onCompleted: (_, errors) =>
-        isEmpty(errors) ? this.createBidder() : this.presentErrorResult(errors, msgForGraphQLError),
-      onError: this.presentErrorResult.bind(this),
+        isEmpty(errors)
+          ? this.createBidder()
+          : this.presentRegistrationResult(RegistrationStatus.RegistrationStatusError),
+      onError: error => {
+        this.presentRegistrationError(error, RegistrationStatus.RegistrationStatusError)
```

Comment:
@yuki24 one question here-- should this be showing the network error screen or the generic error screen?

Reply:
I'm ok with either - my assumption is networrk errors are the majority of the errors that actually occur so the network error screen may be appropriate, but that's still an assumption. I think we should just monitor errors closely once released and iterate quickly on improving the error case scenarios.